jaxrs/src/main/java/com/cerner/beadledom/jaxrs/provider/MalformedRequestFilter.java

```diff
@@ -0,0 +1,37 @@
+package com.cerner.beadledom.jaxrs.provider;
+
+import com.cerner.beadledom.json.common.model.JsonError;
+import java.net.URI;
+import javax.annotation.Priority;
+import javax.ws.rs.Priorities;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import javax.ws.rs.container.PreMatching;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.Provider;
+
+/**
+ * The MalformedRequestFilter reads the request context and determines if the request has a valid uri structure if it
+ * does not then it aborts the request and responds with a 400
+ *
+ * @author Kyle Roush
+ */
+@Provider
+@Priority(Priorities.AUTHENTICATION)
```

Comment:
Could we lower this to a USER priority?

Reply:
Does this error occur at the first call of UriInfo in the stack?